src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.cs

```diff
@@ -132,6 +134,11 @@ public DependencyPropertyDetails GetPropertyDetails(DependencyProperty property)
 
 		private DependencyPropertyDetails? TryGetPropertyDetails(DependencyProperty property, bool forceCreate)
 		{
+			if (_entries is null)
+			{
+				throw new InvalidOperationException("DependencyPropertyDetailsCollection is accessed after it was disposed.");
+			}
```

Comment:
Let's see how this fares, but it may be best to return null instead, or to only fail in debug.